CGPs/cgp-0063/mainnet.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+      "contract": "GasPriceMinimum",
+      "function": "setGasPriceMinimumFloor",
+      "args": [
+        "2200000000000000"
```

Comment:
Data in the proposal body is 25000000000.

Reply:
changed .md data accordingly